### DISCUSSION TITLE
[Projects] OR filter for multiple different fields (bug or FR?)

### DISCUSSION THREAD

**Discussion by Chinoman10** [community/community#36703] [Projects and Issues] [answered]
The OR filter (comma) only works for multiple values of the same field, not for different fields.
Example:
![image](https://user-images.githubusercontent.com/8300763/196780484-0798c7f8-397d-42d6-8ce0-0017e8b612ea.png)

This one ☝️  is filtering as an AND.

While this one 👇  should work:
![image](https://user-images.githubusercontent.com/8300763/196780634-f522f7f0-8eea-4548-bf8e-cb10623cc846.png)

But in reality, it only works for the first status and the 2nd one is ignored.

PS: the objective is to have a single view where all Blocked tasks are visible to an overseeing Ops/Eng. manager(s).
And no, using a "blocked" label isn't a solution since the teams want to have a column for that Status.

**Comment by ossanna16** [comment]
Hey there @Chinoman10 and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by 2percentsilk** [comment, marked as answer]
`OR` filtering across fields is not currently supported but it's on our backlog!
